deltachat-ios/Assets.xcassets/ic_scrolldown.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
```

Comment:
i suggest to add suffixes to the filenames matching their resolution, as `_x1.png`, `_x2.png` , `_x3.png` - see the other assets to get an idea.